dpsim-villas/examples/cxx/CMakeLists.txt

```diff
@@ -33,6 +32,7 @@ set(SHMEM_SOURCES
 )
 
 if(WITH_CIM)
+	list(APPEND SHMEM_SOURCES Fpga9BusHil.cpp)
```

Comment:
Why is the variable called 'SHMEM_SOURCES'? Hasn't the name changed since the shared memory interface?

Reply:
Yes, thats a relict from early times. We should rename this variable..

I will push another commit later..